src/index.ts

```diff
@@ -128,15 +145,15 @@ export class ContentfulLivePreview {
       return null;
     }
 
-    if (!fieldId || !entryId || !locale) {
-      debug.warn('Missing property for inspector mode', { fieldId, entryId, locale });
+    if (!fieldId || !entryId) {
+      debug.warn('Missing property for inspector mode', { fieldId, entryId });
       return null;
     }
 
     return {
       [TagAttributes.FIELD_ID]: fieldId,
       [TagAttributes.ENTRY_ID]: entryId,
-      [TagAttributes.LOCALE]: locale,
+      [TagAttributes.LOCALE]: locale ? locale : this.locale,
```

Comment:
we would only need to render it here if a custom locale is provided, the inspectorMode class could get the locale from the init function and use it as fallback if the tag is not there 🤔

Reply:
do you mean at this point?  https://github.com/contentful/live-preview/blob/155a1483b19b92e35b551d8530b9381901c701ca/src/inspectorMode.ts#L73

Reply:
yes 👍